modules/ROOT/pages/linking.adoc

```diff
@@ -135,6 +134,61 @@ ordersByOrderId (orderId: “123”) {
 }
 --
 
+== Creating a Link Between Object Types Using Composite Keys
+
+The default query method for a type can also use _composite keys_.
+
+Consider a target type `Customer`, which uses composite keys for its default query method: `customerId` and `customerStatus`.
+
+[source]
+--
+customerDetail (id: string!, status: string!): Customer
+--------------------
+--------------------
+Customer
+--------------------
+customerDetail: string!, string!
+customerId: String!
+customerStatus: String!
+--
+
+Assuming the same `Order` schema from above:
+
+[source]
+--
+ordersByOrderId (orderId:String!):Order
+--------------------
+--------------------
+Order
+--------------------
+orderId: String!
+Items: Item[]
+customerId: String!
+--
+
+If you linked the `Customer` type as a target for the `Order` type, the unified schema reflects the link as follows:
+
+[source]
+--
+ordersByOrderId (orderId:String!):Order
+--------------------
+--------------------
+Order
+--------------------
+orderId: String!
+Items: Item[]
+customerId: Customer
+customerStatus: Customer
+--------------------
+--------------------
+Customer
+--------------------
+--------------------
+customerDetail: string!, string!
```

Comment:
This `string` must be `String` because is a Type like as the next fields

Reply:
Fixed, thanks